compiler/src/iree/compiler/Codegen/LLVMCPU/test/pipeline_tests.mlir

```diff
@@ -358,3 +358,41 @@ func.func @unsupported_ukernel_fallback_to_vectorization() attributes {hal.execu
 }
 // CHECK-LABEL: func.func @unsupported_ukernel_fallback_to_vectorization
 // CHECK:         vector.fma
+
+// -----
+
+#pipeline_layout = #hal.pipeline.layout<bindings = [
+  #hal.pipeline.binding<storage_buffer>,
+  #hal.pipeline.binding<storage_buffer>,
+  #hal.pipeline.binding<storage_buffer>
+]>
+#config = #iree_codegen.lowering_config<tile_sizes = [[1, 2, 0, 0, 0, 0], [1, 1, 0, 1, 128, 0], [0, 0, 1, 0, 0, 1]]>
```

Comment:
Why do we have the preset configs and translation_info? Can we remove them, so it also tests the strategy selection?

Reply:
I'll remove them.